Sources/Xcodeproj/pbxproj().swift

```diff
@@ -179,7 +179,14 @@ public func pbxproj(srcroot: AbsolutePath, projectRoot: AbsolutePath, xcodeprojP
         print("        \(module.targetReference) = {")
         print("            isa = PBXNativeTarget;")
         print("            buildConfigurationList = \(module.configurationListReference);")
-        print("            buildPhases = (\(module.compilePhaseReference), \(module.linkPhaseReference));")
+
+        print("            buildPhases = (")
```

Comment:
@abertelrud this part became ugly but I am hoping will become better after your refactor